ops/ansible/environments/demo/group_vars/web.yml

```
@@ -3,4 +3,4 @@ tls_enabled: true
 initdata_src: "{{ inventory_dir }}/assets/initdata.yml"
 
 client_url: "https://demo.catalogue.multi.coop"
-datapass_url: "https://auth.api.gouv.fr"
+datapass_url: "https://auth-demo.api.gouv.fr"
```

Les client id/secret correspondent à l'instance staging donc tu voulais sûrement écrire ceci :

```suggestion
datapass_url: "https://auth-staging.api.gouv.fr"
```